### PR TITLE
Profiles: Fix ENS NFT regressions from merge

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-####

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/parsers/uniqueTokens.js [hunk 1]
```diff
@@ -36,23 +37,17 @@ const parseLastSalePrice = lastSale =>
...
 
-const handleAndSignImages = (
-  contractAddress,
-  imageUrl,
-  previewUrl,
-  originalUrl
-) => {
+const handleAndSignImages = (imageUrl, previewUrl, originalUrl) => {
+  if (!imageUrl && !previewUrl && !originalUrl) {
+    return { imageUrl: undefined, lowResUrl: undefined };
```
Comment (estebanmino) [inline src/parsers/uniqueTokens.js:42]: 🤝

File: src/redux/uniqueTokens.ts [hunk 2]
```diff
@@ -277,30 +279,7 @@ export const fetchUniqueTokens = (showcaseAddress?: string) => async (
...
-                uniqueId: name,
-              };
-            } catch {
-              return token;
-            }
-          }
-          return token;
-        })
-      );
+      newPageResults = await applyENSMetadataFallbackToTokens(newPageResults);
```
Comment (estebanmino) [inline src/redux/uniqueTokens.ts:282]: wondering we should filter ENS here just for readability, this is taking all nfts right? even if there aren't ENS

**Review by estebanmino — APPROVED** [review]
🐐

File: src/redux/uniqueTokens.ts (hunk 2 above)
Comment (jxom) [reply src/redux/uniqueTokens.ts:282]: Won't it need to take all NFTs anyway too check if it is ENS or not?

Comment (jxom) [reply src/redux/uniqueTokens.ts:282]: I thought the "apply fallback" terminology was good enough, but can just do the `.map` here instead

**Pull request merged**